style.css

```diff
@@ -154,7 +157,7 @@ input {
   transition: 0.3s;
   border-radius: 2px;
 }
-.addBtn:hover {
+#add-btn:hover {
```

Comment:
folosim id-uri pentru identificarea elementelor si class pentru stilizare in css